frontend/src/locales/scripts/en.json5

```diff
@@ -930,19 +930,19 @@
   },
   audioTrack: {
     ariaLabel: "Audio: {title}",
-    ariaLabelInteractive: "Audio: {title} - interactive player - press the space bar to play and pause a preview of the audio",
-    ariaLabelInteractiveSeekable: "Audio: {title} - interactive player - press the space bar to play and pause a preview of the audio; use the left and right arrow keys to seek through the track.",
+    ariaLabelInteractive: "Audio: {title} - interactive player - press the space bar to play and pause a preview of the audio track",
+    ariaLabelInteractiveSeekable: "Audio: {title} - interactive player - press the space bar to play and pause a preview of the audio track; use the left and right arrow keys to seek through the track.",
     messages: {
       err_aborted: "You aborted playback.",
       err_network: "A network error occurred.",
-      err_decode: "Could not decode audio.",
+      err_decode: "Could not decode audio track.",
       err_unallowed: "Reproduction not allowed.",
       err_unknown: "An unexpected error has occurred. Try again in a few minutes or report the item if the issue persists.",
-      err_unsupported: "This audio format is not supported by your browser.",
+      err_unsupported: "This audio track format is not supported by your browser.",
       loading: "Loading...",
     },
     creator: "by {creator}",
-    close: "Close the audio player",
+    close: "Close the audio track player",
```

Comment:
Please revert this change as "audio player" is the right word to refer to the player which plays an audio track.